chia/cmds/show.py

```diff
@@ -229,19 +229,24 @@ async def show_async(
 )
 @click.option("-s", "--state", help="Show the current state of the blockchain", is_flag=True, type=bool, default=False)
 @click.option(
-    "-bh", "--block-header-hash-by-height", help="Look up a block header hash by block height", type=str, default=""
+    "-h", "--block-header-hash-by-height", help="Look up a block header hash by block height", type=str, default=""
```

Comment:
breaking this backwards compat should be avoided (I understand the multi-letter options are a problem, but we shouldn't just break things)